python/runtime/common/py_ObserveResult.cpp

```diff
@@ -20,76 +20,104 @@ namespace cudaq {
 void bindObserveResult(py::module &mod) {
   py::class_<observe_result>(
       mod, "ObserveResult",
-      "A data-type containing the results of a call to :func:`observe`. "
-      "This includes any measurement counts data, as well as the global "
-      "expectation value of the user-defined `spin_operator`.\n")
+      """
+      A data-type containing the results of a call to :func:`observe`
+      This includes any measurement counts data, as well as the global 
+      expectation value of the user-defined `spin_operator`
+      """
+      )
       /// @brief Bind the member functions of `cudaq.ObserveResult`.
       .def("dump", &observe_result::dump,
-           "Dump the raw data from the :class:`SampleResult` that are stored "
-           "in :class:`ObserveResult` to the terminal.\n")
+            """
+            Dump the raw data from the :class:`SampleResult` that are stored
+            in :class:`ObserveResult` to the terminal.
+            """
+           )
       .def(
           "counts", &observe_result::raw_data,
-          "Returns a :class:`SampleResult` dictionary with the measurement "
-          "results from the experiment. The result for each individual term of "
-          "the `spin_operator` is stored in its own measurement register. "
-          "Each register name corresponds to the string representation of the "
-          "spin term (without any coefficients).\n")
+          """
+          Returns:
+          :class:`SampleResult` dictionary with the measurement 
+          results from the experiment. 
+          The result for each individual term of the `spin_operator` is stored 
+          in its own measurement register. 
+          Each register name corresponds to the string representation of the 
+          spin term (without any coefficients).
+          """
+          )
       .def("counts", &observe_result::counts<spin_op>, py::arg("sub_term"),
-           "Given a `sub_term` of the global `spin_operator` that was passed "
-           "to :func:`observe`, return its measurement counts.\n"
-           "\nArgs:\n"
-           "  sub_term (:class:`SpinOperator`): An individual sub-term of the "
-           "`spin_operator`.\n"
-           "\nReturns:\n"
-           "  :class:`SampleResult` : The measurement counts data for the "
-           "individual "
-           "`sub_term`.\n")
+            """
+            Given a `sub_term` of the global `spin_operator` that was passed 
+            to :func:`observe`, return its measurement counts.
+            
+            Args:
+            sub_term (:class:`SpinOperator`): An individual sub-term of the
+            `spin_operator`.
+            
+            Returns:
+            :class:`SampleResult` : The measurement counts data for the 
+            individual `sub_term`.
+            """
```

Comment:
And this is the counterexample. A big block of text with lots of formatting where raw strings can improve readability.